src/main/java/uk/gov/pay/api/model/PaymentState.java

```diff
@@ -4,20 +4,25 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.JsonNode;
+import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
+@ApiModel(value="Payment state", description = "A structure representing the current state of the payment in its lifecycle.")
 public class PaymentState {
     @JsonProperty("status")
     private String status;
 
+    @ApiModelProperty(value = "Whether the payment has finished", required = true)
```

Comment:
I think the `@ApiModelProperty` annotations here are overridden by the ones in the associated annotation ion the getter method. better to remove these and use only the one around getter.